src/test/java/de/mediathekview/mserver/testhelper/AssertFilm.java

```diff
@@ -0,0 +1,54 @@
+package de.mediathekview.mserver.testhelper;
+
+import de.mediathekview.mlib.daten.Film;
+import de.mediathekview.mlib.daten.GeoLocations;
+import de.mediathekview.mlib.daten.Resolution;
+import de.mediathekview.mlib.daten.Sender;
+import java.net.URL;
+import java.time.Duration;
+import java.time.LocalDateTime;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.notNullValue;
+import org.hamcrest.Matchers;
+import static org.junit.Assert.assertThat;
+
+public class AssertFilm {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)